Planner/app/src/main/java/rdatu/android/cyscorpions/com/projectplanner/model/TasksDatabaseHelper.java

```diff
@@ -0,0 +1,137 @@
+package rdatu.android.cyscorpions.com.projectplanner.model;
+
+import android.content.ContentValues;
+import android.content.Context;
+import android.database.Cursor;
+import android.database.CursorWrapper;
+import android.database.sqlite.SQLiteDatabase;
+import android.database.sqlite.SQLiteOpenHelper;
+import android.util.Log;
+
+import java.util.ArrayList;
+
+/**
+ * Created by rayeldatu on 7/28/15.
+ */
+public class TasksDatabaseHelper extends SQLiteOpenHelper {
+
+    private static String DATABASE_NAME = "task.sqlite";
+    private static String TABLE_TASKS = "tasks";
+    private static String COLUMN_TASK_ID = "_id";
+    private static String COLUMN_TASK_NAME = "task_name";
+    private static String COLUMN_TASK_DESC = "task_description";
+    private static String COLUMN_PLACE = "task_place";
+    private static String COLUMN_DATE = "task_date";
+    private static String COLUMN_TIMESLOT = "task_time";
+    private static String COLUMN_PRIORITY = "task_priority";
+    private static SQLiteDatabase mDatabase;
+
+    public TasksDatabaseHelper(Context context) {
+        super(context, DATABASE_NAME, null, 1);
+    }
+
+
+    @Override
+    public void onCreate(SQLiteDatabase db) {
+        db.execSQL("create table tasks (task_date varchar(30),task_time varchar(15), task_name varchar(100), task_description varchar(200), task_place varchar(100),task_priority varchar(10))");
+        mDatabase = db;
+    }
+
+    public long insertTasks(String name, String desc, String place, String date, String time, String priority) {
+        ContentValues cv = new ContentValues();
+        cv.put(COLUMN_DATE, date);
+        cv.put(COLUMN_TIMESLOT, time);
+        cv.put(COLUMN_TASK_NAME, name);
+        cv.put(COLUMN_TASK_DESC, desc);
+        cv.put(COLUMN_PLACE, place);
+        cv.put(COLUMN_PRIORITY, priority);
+        return getWritableDatabase().insert(TABLE_TASKS, null, cv);
```

Comment:
We can have 1 instance of getWritableDatabase() initialized in the constructor of this class instead of calling this method each time we want to manipulate data